### DISCUSSION TITLE
Error in Jupyter Notebook and Pycharm

### DISCUSSION THREAD

**Discussion by nupadhayay54** [datasciencecompleteguide/Machine-Learning#19] [Q&A]
cv2.imshow is not working in jupyter notebook whereas plt.imshow worked.
plt.imshow is not working in pycharm whereas cv2.imshow worked.

Also suggest the IDE in which both cv2.imshow and plt.imshow worked.

**Comment by datasciencecompleteguide** [comment]
Hey, Nothing Like that both the things work on both the idle. Although, there are different methods for implementing both the things.

In the Image below, I have used the plt.imshow() in my pycharm.
Also, plt.imshow() uses the BGR as its default color code.
![Screenshot (47)](https://user-images.githubusercontent.com/69833825/128287141-98bd9405-37ae-484d-910a-4520f4d7a88d.png)

In the Image below, I have used the cv2.imshow() in my jupyter notebook.
![Screenshot (48)](https://user-images.githubusercontent.com/69833825/128287266-548ec653-ee2b-43a7-b637-d009a54ff52f.png)
